templates/strapdown.html.hbs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<title>API Document</title>
```

Comment:
This template doesn't conform to the standard that most all HTML documents follow on the web. The format should be the following:

```html
<!DOCTYPE html>
<html>
  <head>
    <title></title>
  </head>
  <body>
    {{ content }}
    {{ scripts to load asynchronously }}
  </body>
</html>
```

Reply:
Unfortunately the js code gets buggy if I try it that way. How important is the formatting?